app/src/organisms/CommandText/PipettingCommandText.tsx

```diff
@@ -61,7 +57,7 @@ export const PipettingCommandText = ({
       const { volume, flowRate } = command.params
       return t('aspirate', {
         well_name: wellName,
-        labware: getLabwareName(robotSideAnalysis, labwareId),
+        labware: getLabwareName(robotSideAnalysis, labwareId ?? ''),
```

Comment:
I think this extra protection isn't needed since labwareId is already specified to be a string up above 

```suggestion
        labware: getLabwareName(robotSideAnalysis, labwareId),
```